### ISSUE TITLE
SASL support to log into irc.libera.chat

### ISSUE THREAD

**Issue by Endle** [RITlug/teleirc#385]
# Summary

I'm trying to run teleirc on my server. I used the dockerfile to run it.

```
docker run \
    -e IRC_SERVER=irc.libera.chat \
    -e IRC_CHANNEL="#fedora-zh" \
    -e IRC_BOT_IDENT=teleirc \
    -e IRC_PORT=8000 \
    -e IRC_BOT_NAME="fedora_zh_bridge_bot" \
    -e IRC_NICKSERV_SERVICE=NickServ \
    -e IRC_NICKSERV_USER="fedora_zh_bridge_bot" \
    -e TELEGRAM_CHAT_ID="-0000000000" \
    -e IRC_NICKSERV_PASS="***********" \
    -e TELEIRC_TOKEN="="***********":"***********"" \
	ghcr.io/endle/teleirc_git:1634289171_7382f25
```

# Outcome
I failed to log-in libra.chat
```
INFO: 2021/10/15 10:14:39 Current TeleIRC version: 
INFO: 2021/10/15 10:14:39 Creating new Telegram bot client...
INFO: 2021/10/15 10:14:39 Creating new IRC bot client...
INFO: 2021/10/15 10:14:39 Starting up Telegram bot...
INFO: 2021/10/15 10:14:39 Starting up IRC bot...
INFO: 2021/10/15 10:14:39 Authorized on account fedora_zh_bridge_bot
ERROR: 2021/10/15 10:14:39 closing connection: SASL authentication failed
ERROR: 2021/10/15 10:14:39 closing connection: SASL authentication failed
INFO: 2021/10/15 10:14:39 Shutting Down...
INFO: 2021/10/15 10:14:39 Exiting

```

It might be caused by https://libera.chat/guides/sasl#sasl-access-only-ip-ranges

**Comment by Tjzabel** [comment]
Hello,

I'm going to assume the issue is with your `IRC_PORT=8000`, as this should most likely be `=6697` as that's what Libera uses for SASL. 

Can you please try making that change as see if that fixes the issue?

**Comment by jwflory** [comment]
Hi @Endle, were you able to re-test this with port `6697`?

**Comment by Endle** [comment]
Sorry for my delay. I should provide my test result earlier.

Before testing the 6697 port, I tried `8000` port again. This time, I couldn't reproduce my test result in https://github.com/RITlug/teleirc/issues/385#issue-1027304298

Nothing changed on my client (and I'm using a Docker). I can only say that some changes happened on libra.chat (maybe they moved my IP out of their block list)

I think it confirmed my first thought that this is not a bug in teleirc (that's why I tagged this issue as feature instead of bug).
